model/textparse/openmetricsparse_test.go

```diff
@@ -992,3 +994,52 @@ go_gc_duration_seconds_created`)
 	require.Equal(t, "go_gc_duration_seconds", string(copyParser.l.b[copyParser.offsets[0]:copyParser.offsets[1]]))
 	require.False(t, copyParser.skipCTSeries)
 }
+
+func BenchmarkOMParse(b *testing.B) {
+	for parserName, parser := range map[string]func([]byte, *labels.SymbolTable) Parser{
+		"openmetrics": func(b []byte, st *labels.SymbolTable) Parser {
+			return NewOpenMetricsParser(b, st)
+		},
+		"openmetrics-skip-ct": func(b []byte, st *labels.SymbolTable) Parser {
+			return NewOpenMetricsParser(b, st, WithOMParserCTSeriesSkipped())
+		},
+	} {
+		f, err := os.Open("omtestdata.txt")
+		require.NoError(b, err)
+		defer f.Close()
+
+		buf, err := io.ReadAll(f)
+		require.NoError(b, err)
+
+		b.Run(parserName+"/parse-ct/"+"omtestdata.txt", func(b *testing.B) {
+			if !strings.HasPrefix(parserName, "openmetrics") {
+				b.Skip()
+			}
+			b.SetBytes(int64(len(buf) / promtestdataSampleCount))
+			b.ReportAllocs()
+			b.ResetTimer()
+
+			total := 0
```

Comment:
actually, let me ask one thing.

I noticed that you copied this part from the previous benchmark, but do we know what `total` is used for? It's not obvious to me at first glance, maybe not needed here?

Reply:
I actually am curious as to what its used for 
but yeah don't need this here

Reply:
removed now